analyses/cell-type-ewings/README.md

```diff
@@ -11,6 +11,10 @@ Currently, we use a variety of methods to annotate tumor cells independently:
 The full list of marker genes can be found in `references/tumor-marker-genes.tsv`.
 - Use the list of tumor marker genes to classify tumor cells with [`CellAssign`](https://docs.scvi-tools.org/en/stable/user_guide/models/cellassign.html).
 - Identify copy number variations and annotate tumor cells using [`CopyKAT`](https://github.com/navinlabcode/copykat).
+- Identify copy number variations using [`InferCNV`](https://github.com/broadinstitute/inferCNV/wiki).
+This returns a proportion of each chromosome with a CNV detected.
+We then calculate the mean proportion for each cell across all chromsomes and weight by the number of genes in a chromosome.
+Cells with a mean proportion > 0.1 are called as tumor cells.
```

Comment:
I think this is fine for now, but I do wonder if there may be a way to find a split in a bimodal distribution, should one exist. (I have not yet gotten to the data... just thinking ahead)

Reply:
Added a thought below that maybe just the mean is the way to do this

Reply:
So just the mean of the scaled proportion values across all chromosomes rather than weighting by chromosome? Or just taking the mean of the sum of total chromosomes with CNVs?

Reply:
Two different answers, which are really the same:
- For using the total chromosomes with CNVs, just the mean # of chromosomes with CNVs as the dividing line. (This is the same as scaling and then using 0).
- For the weighted mean proportion CNV values, again using the mean across cells as the dividing line. The idea here is that if there is a bimodal distribution, that should generally lie _somewhere_ between the two peaks. There are going to be more sophisticated ways of choosing a dividing line, but I expect this to be a reasonable place to start.